README.md

```diff
@@ -149,7 +149,34 @@ translations_list = [
     ('de_DE', 'German'),
 ]
 
-content_prefix = "ecosystem/finance"
+docs_url_prefix = "ecosystem/finance"
+```
+
+## Enable Previous Releases
+
+This feature allows you to link to previous versions of the docs in the left sidebar.
+
+First, start additionally deploying your docs to `<project-prefix>/stable/<version>/`, e.g. `/ecosystem/finance/stable/0.5/index.html`. See https://github.com/Qiskit/qiskit-experiments/blob/227867937a08075092cd11756214bee3fb1d4d3d/tools/deploy_documentation.sh#L38-L39 for an example.
+
+Then, update your `conf.py`:
+
+* Ensure that `qiskit_sphinx_theme` is in the `extensions` setting.
+* Add to the option `html_context` an entry for `version_list` with a list of the prior versions, e.g. `["0.4", "0.5"]`. Each of these versions must be deployed with the above `stable/<version>` URL scheme.
```

Comment:
Some projects use an auto-generated list: https://github.com/Qiskit/qiskit-experiments/blob/a8f9de8b39774ce2c22c0ec5ae19bd7f023bfeca/docs/conf.py#L185 

Which method should be listed here as the standard?

Reply:
I want to investigate in https://github.com/Qiskit/qiskit_sphinx_theme/issues/307 if we can get more standardization for Previous Releases. In the meantime, it's up to each project to do this how they want.

I rewrote this section to give more context. Thanks for the feedback!